<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Goods</class>
 <widget class="QDialog" name="Goods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Select kind:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_kind"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string> Select unit:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_unit"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Set price :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_price"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Enter item:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_additem"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_additem">
     <property name="text">
      <string>Add item</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
